bigchaindb/voting.py

```diff
@@ -0,0 +1,152 @@
+import collections
+
+from bigchaindb.common.schema import SchemaValidationError, validate_vote_schema
+from bigchaindb.common.utils import serialize
+from bigchaindb.common.crypto import PublicKey
+
+
+VALID = 'valid'
+INVALID = 'invalid'
+UNDECIDED = 'undecided'
+
+
+class Voting:
```

Comment:
all methods here are `classmethod`s, would it make sense to remove the class and just have functions?